easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.6-goolf-1.7.20-MPI.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = "CMakeMake"
+
+name = 'IQ-TREE'
+version = '1.5.6'
+versionsuffix = '-MPI'
+
+homepage = 'http://www.iqtree.org'
+description = "A fast and effective stochastic algorithm to infer phylogenetic trees by maximum likelihood"
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
```

Comment:
@pescobar No `toolchainopts = {'usempi': True}` needed?

Reply:
Also, we had to disable architecture-specific optimisations (`'optarch': False`) in #3695...

How have you built `IQ-TREE`, any related `--optarch` settings you used?